packages/blockchain-wallet-v4-frontend/src/modals/components/index.tsx

```diff
@@ -435,13 +435,14 @@ const StyledOvalButton = styled(Button)`
   border-radius: 32px;
 `
 
-const AlertButton = ({ children }) => (
+const AlertButton = ({ children, onClick = () : void => {}}) => (
```

Comment:
I think you might be putting wrong the default value for onClick, also it may not be necessary. 
In the typing it should be () => void in case of no response

Reply:
i agree, no need to add a default here.

Reply:
If I don't add the default value for onClick it forces me to add the property on all AlertButtons. I just need (for now) one AlertButton with onClick and the others without the property. Any idea?

<img width="1150" alt="Captura de Pantalla 2022-03-21 a la(s) 19 59 09" src="https://user-images.githubusercontent.com/97241711/159376829-2ed957cb-2b72-4e81-8d97-2803f7f2131e.png">